frontend/src/screens/dashboard/docsqa/main/partials/Form.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import Input from '@/components/base/atoms/Input'
+import Button from '@/components/base/atoms/Button'
+
+const Form = (props: any) => {
```

Comment:
this component name is confusing ... Form represents an atomic component ... this could be PromptForm or something like that